src/backend/libs/efiling-account-client/src/main/java/ca/bc/gov/open/jag/efilingaccountclient/CsoAccountDetails.java

```diff
@@ -0,0 +1,36 @@
+package ca.bc.gov.open.jag.efilingaccountclient;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class CsoAccountDetails {
+
+    CsoAccountDetails(String accountId, String clientId, String[] roles) {
+
+        this.accountId = accountId;
+        this.clientId = clientId;
+
+        for (String role: roles) {
+            String toAdd = role.toLowerCase();
+            this.roles.add(toAdd);
+        }
+    }
+
+    private String accountId;
+    private String clientId;
+    private List<String> roles;
```

Comment:
should we instantiate the list ?

```suggestion
private List<String> roles = new ArrayList<>();
```